src/backend/api/Fusion.Resources.Api/Authorization/Handlers/ContractRoleAuthHandler.cs

```diff
@@ -18,24 +18,24 @@ public ContractRoleAuthHandler(IProjectOrgResolver orgResolver)
 
         protected override async Task HandleRequirementAsync(AuthorizationHandlerContext context, ContractRole requirement, ContractResource resource)
         {
-            var contract = await orgResolver.ResolveContractAsync(resource.Project.ProjectId, resource.Contract);
+            var contract = await orgResolver.ResolveContractAsync(resource.ProjectId, resource.Contract);
             if (contract == null)
             {
-                requirement.SetEvaluation($"Couldn't locate contract in project '{resource.Project.Name}'");
+                requirement.SetEvaluation($"Couldn't locate contract in project '{resource.ProjectId}'");
```

Comment:
Might be relevant to have Project.Name ?
Name do not exist in Fusion.Integration.Authorization.